### PR TITLE
update to 14.0.6

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Comment by katietz** [comment]
The Windows build seems to fail 6by missing installation of AtlBase in VS2019 context.  At least it looks that way by reading threads on llvm on Windows with same error. The DIA-sdk is found.
I will ask DK on the installation settings of VS2019.  Not sure, but maybe VS2017 would be worth a try instead. It could just be that other components in llvm might be no longer compatible to it. And it is not clear if ATL-Base is installed at all for it

**Comment by katietz** [comment]
I could built it successful on Windows using concourse, so it is a prefect-builder issue on Windows.  See https://concourse.build.corp.continuum.io/teams/main/pipelines/ktietz_t1

**Comment by AndrewVallette** [comment]
Due to the way LLVM implements support for DIA SDK, the `win-64` `llvmdev` package here still contains a hard-coded, machine-specific path to the SDK, and this will cause `llvmlite` to fail to build with this package on Prefect, because DIA is located in a different subdir.  `INTERFACE_LINK_LIBRARIES` ideally should not contain this sort of path (see the generated LLVMExports.cmake). The patch I'm testing (for 11.1) should fix this here as well.

**Review by varlackc — COMMENTED** [review]
https://github.com/AnacondaRecipes/llvmdev-feedstock/blob/77470ab9e91f8d236f2e733f4a0fcf576a9c9606/recipe/meta.yaml#L173

might need to use `https` instead of `http` for the `home url`
https://llvm.org/

We might need a `doc_url` the following might be a good candidate for the documentation:
https://llvm.org/docs/

**Comment by katietz** [comment]
thank you all for your suggestion.  I address them all

**Review by cbouss — COMMENTED** [review]
LGTM. 
abs.yaml could have different options, but this is more nice to have than must change.

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# the conda-build parameters to use for disabling --skip-existing
+build_parameters:
+  - ""
```
[the remark this answers is not in the document]
Comment (katietz) [reply abs.yaml:3]: discussed in chat. maybe later

**Pull request merged**